telegapi/utils.py

```diff
@@ -33,10 +33,14 @@ def msg_get_summary(msg, truncate=0):
     summary = ""
 
     if 'text' in msg:
-        summary += "[Text: " +\
-                   ((msg["text"][:truncate] + '...')
-                    if (len(msg["text"]) > truncate and truncate is not 0)
-                    else msg["text"]) + "] "
+
+        message = msg["text"]
+        if (0 < truncate < len(message)):
+            summary = message[:truncate] + '...'
+        else:
+            summary = message + ']'
+
+        summary = "[Text: " + str(summary.encode('unicode_escape'))
```

Comment:
There is the `unicodedata` module, which does something similar:
```python
uni.name('\U0001f648')
# => 'SEE-NO-EVIL MONKEY'
```

Things is, all python 3 strings are unicode, and the precise emoji range is pretty loose. I'll see if I can come up with something better in the meantime